docs/GettingStarted.md

```diff
@@ -6,7 +6,9 @@ Before using the AWS Advanced JDBC Driver, you must install:
 
 - Amazon Corretto 8+ or Java 8+.
 - The AWS Advanced JDBC Driver.
-- Your choice of underlying JDBC driver. To use the wrapper with Aurora with PostgreSQL compatibility, you need to install the [PostgreSQL JDBC Driver](https://github.com/pgjdbc/pgjdbc).
+- Your choice of underlying JDBC driver. 
+  - To use the wrapper with Aurora with PostgreSQL compatibility, install the [PostgreSQL JDBC Driver](https://github.com/pgjdbc/pgjdbc).
+  - To use the wrapper with Aurora with MySQL compatibility, install the [MySQL JDBC Driver](https://github.com/mysql/mysql-connector-j).
```

Comment:
Does the MariaDB driver need to be added here?

Reply:
added!